pipelines/controller/src/test/java/org/openmrs/analytics/metrics/BaseApplicationTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.TestPropertySource;
+
+@SpringBootTest
+@TestPropertySource("classpath:application-test.properties")
+public class BaseApplicationTest {
+
+  @TestConfiguration
+  public static class TestAppConfig {
+
+    @Bean
+    public MeterRegistry getMeterRegistry() {
```

Comment:
Why do we need to redefine this? I mean can't we use the same `@Bean` method in `AppConfig` above?